CONTRIBUTING.md

```diff
@@ -100,6 +100,29 @@ The packages are available at the following urls with hot-reloading:
 - Front-end: http://localhost:3000
 - Back-end: http://localhost:3100
 
+#### Accessing the MongoDB database
+
+To access MongoDB with the [mongosh shell](https://www.mongodb.com/docs/mongodb-shell/), run the following command:
```

Comment:
Consider adding the following so people don't lose their section in the docs.
`<a href="https://www.mongodb.com/docs/mongodb-shell/" target="_blank">mongo shell</a>`

Reply:
Thanks for the feedback! I would recommend not implementing that suggestion for the following reasons:

- consistency with existing links (none of the other 20+ links open in new tabs)
- accessibility

To elaborate on the accessibility point, it's recommended for accessibility purposes not to open a link in a new window by default, and the user can press Cmd/Ctrl while clicking to do so on their own accord. Here's a few articles on the topic:

- WCAG 2.0 accessibility guidelines: https://www.w3.org/TR/WCAG20-TECHS/G200.html
- some agency specializing in accessibility: https://scaramanga.agency/articles/2021/do-not-open-links-in-a-new-tab-or-window

Reply:
Interesting, good to know.